app/Http/Controllers/Skin/SkinGetBody.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers\Skin;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Skin\SkinUtils;
+use App\Http\Requests\Skin\SkinBodyRequest;
+
+class SkinGetBody extends Controller
+{
+    /**
+     * Handle the incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function __invoke(SkinBodyRequest $request)
+    {
+        $uuid = $request->uuid;
+        $scale = $request->scale ?? 1;
+        $gear = $request->gear ?? 0;
+        try {
+            $skin = SkinUtils::getBodyFromUUID($uuid, $scale, $gear);
+        } catch (\Exception $e) {
+            if ($e->getMessage() == "invalid_uuid") {
+                return response()->json([
+                    'errors' => [
+                        'uuid' => [
+                            'The uuid does not seem to be a valid UUID.'
+                        ]
+                    ]
+                ], 422);
+            }
```

Comment:
You probably want to handle other errors

Reply:
https://github.com/redcraft-org/RedCraftWebsiteV3/blob/635e80ed206565488803d67148d85dcb53248419/app/Http/Requests/Skin/SkinBodyRequest.php